packages/hash/api/src/model/util.ts

```diff
@@ -100,14 +107,14 @@ export const generateSchemaBaseUri = (params: {
 }) =>
   `${params.namespaceUri}/${schemaKindSlugs[params.kind]}/${slugifySchemaTitle(
     params.title,
-  )}`;
+  )}/` as const;
 
 export const generateSchemaVersionedUri = (params: {
   namespaceUri: string;
   kind: SchemaKind;
   title: string;
   version?: number;
-}) => `${generateSchemaBaseUri(params)}/v/${params.version ?? 1}` as const;
+}) => `${generateSchemaBaseUri(params)}v/${params.version ?? 1}` as const;
```

Comment:
This change is pretty important as it fixes issues with the workspace in regards to seeding types and users. 

Would have been best if we didn't rely on our own functions here for generating these URIs.